ajax/libs/selectonic/package.json

```diff
@@ -0,0 +1,40 @@
+{
+  "name": "selectonic",
+  "version": "0.6.3",
+  "description": "jQuery-plugin for making any list of items selectable by mouse and keyboard.",
+  "author": {
+    "name": "Alexey Novichkov",
+    "email": "mygrib@gmail.com"
+  },
+  "filename": "selectonic.min.js",
+  "keywords": [
+    "list",
+    "jquery",
+    "selectable",
+    "selection"
+  ],
+  "repository": {
+    "type": "git",
+    "url": "https://github.com/anovi/selectonic.git"
+  },
+  "license": "MIT",
+  "homepage": "http://anovi.github.io/selectonic/",
```

Comment:
Let's use `https`
This website has SSL certificate.
We provide more security homepage info for people to help they browsing safety.
Thank you.

Reply:
@sufuf3 Done!